build/webpack.config.js

```diff
@@ -154,7 +154,7 @@ const PATHS_TO_TREAT_AS_CSS_MODULES = [
 // If config has CSS modules enabled, treat this project's styles as CSS modules.
 if (config.compiler_css_modules) {
   PATHS_TO_TREAT_AS_CSS_MODULES.push(
-    paths.client().replace(/[\^\$\.\*\+\-\?\=\!\:\|\\\/\(\)\[\]\{\}\,]/g, '\\$&')
+    paths.client().replace(/[\^\$\.\*\+\-\?\|\\\/\(\)\[\]\{\}]/g, '\\$&')
```

Comment:
It was marked as a linting error until I made the suggested changes. If you want to keep it as is, I would recommend an eslint ignore line comment. If you aren't getting a linting error it could be an issue with my text editor as I have standard js installed as seperate package along with eslint related packages.